resources/views/pages/game/[game]/comments.blade.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use App\Models\Game;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{middleware, name, render};
+
+middleware(['auth', 'can:viewAny,' . Game::class]);
+name('game.comments');
+
+render(function (View $view, Game $game) {
+    return $view->with([
+        'game' => $game,
+    ]);
+});
+
+?>
+
+@php
+
+use App\Community\Enums\ArticleType;
+
+@endphp
+
+@props([
+    'game' => null, // Game
+])
+
+<x-app-layout pageTitle="Comments: {{ $game->Title }}">
```

Comment:
Since this will be public-facing, it (and other public-facing comments routes) needs a `pageDescription` or we'll get dinged by Google.

Reply:
done